src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java

```diff
@@ -0,0 +1,124 @@
+package io.github.jhipster.jpa2jdl;
+
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+
+import javax.persistence.Entity;
+import java.io.BufferedWriter;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.lang.annotation.Annotation;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+public class NewRunner {
```

Comment:
whats the rationale for a new class called "NewRunner" instead of the old main class? renaming classes and creating unnecessary separation of concerns IMO makes code changeset hard to follow. i understand everybody has their own fav structure/pattern they wanna impose, but lets try to follow the original authors structure if it is simply naming and stylistic change.